### PR TITLE
QA Observation Bugfix/FOUR-13113: The slider lines of the carousel are not the same as figma

### PR DESCRIPTION
**Pull request by CarliPinell**
## How to Test
-  Open Any Process with "Open in Launchpad" and check Carousel Sliders

## Related Tickets & Packages
https://processmaker.atlassian.net/browse/FOUR-13113

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.
ci:deploy
ci:next

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-203ade53b2.eng.processmaker.net

File: resources/js/processes-catalogue/components/ProcessesCarousel.vue [hunk 1]
```diff
@@ -74,7 +74,26 @@ export default {
   },
 };
 </script>
-<style scoped>
+<style lang="scss">
+#carousel-1 {
```
Comment (fagubla) [inline resources/js/processes-catalogue/components/ProcessesCarousel.vue:78]: Change de ID name is so generic

**Review by fagubla — CHANGES_REQUESTED** [review]

File: resources/js/processes-catalogue/components/ProcessesCarousel.vue (hunk 1 above)
Comment (CarliPinell) [reply resources/js/processes-catalogue/components/ProcessesCarousel.vue:78]: Solved. Changed to processes-carousel

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-6582783252.engk8s.processmaker.net

**Review by fagubla — APPROVED** [review]

**Pull request merged**